### PR TITLE
Improve the texts

### PR DESCRIPTION
**Pull request by joachimprinzbach**

### CODE REVIEW

**Comment by incubator-tekton** [comment]
Welcome to Baloise ChatOps based GitOps Bot.<br><br>If you want to enable preview environments for this PullRequest, comment with `/preview`.<br>If you want to disable the preview environment for this PR, comment with `/undeploy`.<br>If you need help or want to see the docs, comment with `/help`.<br><br>At the moment, preview environments do not get redeployed automatically. Simply rewrite `/preview` or contribute the feature.

**Comment by incubator-tekton** [comment]
New Preview Environment for gitops-nginx-demo and branch feat/imporve-texts created successfully. Access it here: 
https://gitops-nginx-demo-641a87da.apps.origin.baloise.dev

**Comment by incubator-tekton** [comment]
Preview Environment for gitops-nginx-demo and branch feat/imporve-texts updated successfully. Access it here: 
https://gitops-nginx-demo-641a87da.apps.origin.baloise.dev

**Comment by incubator-tekton** [comment]
Preview Environment for gitops-nginx-demo and branch feat/imporve-texts updated successfully. Access it here: 
https://gitops-nginx-demo-641a87da.apps.origin.baloise.dev

**Comment by incubator-tekton** [comment]
Preview Environment for gitops-nginx-demo and branch feat/imporve-texts updated successfully. Access it here: 
https://gitops-nginx-demo-641a87da.apps.origin.baloise.dev

**Comment by incubator-tekton** [comment]
Preview Environment for gitops-nginx-demo and branch feat/imporve-texts updated successfully. Access it here: 
https://gitops-nginx-demo-641a87da.apps.origin.baloise.dev

File: static/index.html [hunk 1]
```diff
@@ -23,73 +23,71 @@
...
-                        <p>GitOps, blabla & mehr gedöhns das keiner hören will :D :D :D</p>
+                        <p>From the idea of a new product until the deployment in production - a matter of seconds, only by applying git commands!</p>
                         <ul>
-                            <li>-  Praesent porta urna id eros</li>
-                            <li>-  Curabitur consectetur malesuada</li>
-                            <li>-  Nam pretium imperdiet enim</li>
-                            <li>-  Sed viverra arcu non nisi efficitur</li>
+                            <li>-  Automatically create new namespaces securely</li>
+                            <li>-  Decorate PullRequests using a ChatOps based Bot</li>
+                            <li>-  preview environments for every PullRequest</li>
```
Comment (niiku) [inline static/index.html]: Capital P

**Review by niiku — APPROVED** [review]

**Pull request merged**